docs/quickstarts/insights-malware-install/insights-malware-install.yml

```diff
@@ -0,0 +1,117 @@
+# Additional info: https://docs.openshift.com/container-platform/4.9/web_console/creating-quick-start-tutorials.html
+metadata:
+  name: insights-malware-install
+  # you can add additional metadata here
+  instructional: true
+spec:
+  displayName: Install, test and run the malware detection Service
+  durationMinutes: 10
+  # Optional type section, will display as a tile on the card
+  type:
+    text: Quick start
+    # 'blue' | 'cyan' | 'green' | 'orange' | 'purple' | 'red' | 'grey'
+    color: grey
+  icon: data:image/svg+xml;base64,PCEtLSBHZW5lcmF0ZWQgYnkgSWNvTW9vbi5pbyAtLT4KPHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgd2lkdGg9IjUxMiIgaGVpZ2h0PSI1MTIiIHZpZXdCb3g9IjAgMCA1MTIgNTEyIj4KPHRpdGxlPjwvdGl0bGU+CjxnIGlkPSJpY29tb29uLWlnbm9yZSI+CjwvZz4KPHBhdGggZD0iTTQ0OCA2NHY0MTZoLTMzNmMtMjYuNTEzIDAtNDgtMjEuNDktNDgtNDhzMjEuNDg3LTQ4IDQ4LTQ4aDMwNHYtMzg0aC0zMjBjLTM1LjE5OSAwLTY0IDI4LjgtNjQgNjR2Mzg0YzAgMzUuMiAyOC44MDEgNjQgNjQgNjRoMzg0di00NDhoLTMyeiI+PC9wYXRoPgo8cGF0aCBkPSJNMTEyLjAyOCA0MTZ2MGMtMC4wMDkgMC4wMDEtMC4wMTkgMC0wLjAyOCAwLTguODM2IDAtMTYgNy4xNjMtMTYgMTZzNy4xNjQgMTYgMTYgMTZjMC4wMDkgMCAwLjAxOS0wLjAwMSAwLjAyOC0wLjAwMXYwLjAwMWgzMDMuOTQ1di0zMmgtMzAzLjk0NXoiPjwvcGF0aD4KPC9zdmc+Cg==
+  prerequisites:
+    - The system operating system must be RHEL8 or RHEL9
+    - The administrator must have sudo access on the system
+    - The system must have the Insights client package installed, and be registered to Insights for Red Hat Enterprise Linux.
+  description: |-
+
+    Install and register the malware service
+    
+  introduction: |-
+    Welcome to the quick start that will install YARA and the Insights-client. YARA is an open-source and multi-platform tool that detects code similarities between predefined malware signatures and a scanned RHEL 
+    system. If one or more conditions in a predefined signature exist on a scanned RHEL system, YARA records a malware hit on that system. 
+    Install the YARA Package on each host you want to scan for Malware and Ensure the System’s Operating System versions are either RHEL 8 or RHEL 9.  
```

Comment:
Malware and Ensure capitalized here. Intentional? I would lean toward lowercase, but check all malware for consistent case.